query-engine/driver-adapters/src/conversion.rs

```diff
@@ -75,11 +75,8 @@ pub fn values_to_js_args(values: &[quaint::Value<'_>]) -> serde_json::Result<Vec
                 Some(bytes) => JSArg::Buffer(bytes.to_vec()),
                 None => JsonValue::Null.into(),
             },
-            quaint_value @ quaint::ValueType::Numeric(bd) => match bd {
-                Some(bd) => match bd.to_string().parse::<f64>() {
-                    Ok(double) => JSArg::from(JsonValue::from(double)),
-                    Err(_) => JSArg::from(JsonValue::from(quaint_value.clone())),
-                },
+            quaint::ValueType::Numeric(bd) => match bd {
+                Some(bd) => JSArg::RawString(bd.to_string()),
```

Comment:
Can we leave a comment here briefly explaining the rationale of this line for future reference?